mmdet/core/post_processing/bbox_nms.py

```diff
@@ -31,15 +31,15 @@ def multiclass_nms(multi_bboxes,
         tuple: (bboxes, labels, indices (optional)), tensors of shape (k, 5),
             (k), and (k). Labels are 0-based.
     """
-    num_classes = multi_scores.size(1) - 1
+    num_classes = int(multi_scores.size(1) - 1)
```

Comment:
For which reason do we need to convert this to int here?

Reply:
After exporting the model, the number of classes most likely does not change.
So we can convert this value to a constant. This simplifies the graph in ONNX.

This is how `scores = multi_scores [:,: num_classes]` looks in the graph when `num_classes` is a constant:
<img width="408" alt="model_const" src="https://user-images.githubusercontent.com/10177297/115424739-a44e5580-a207-11eb-81b6-99548a9f3933.PNG">

Otherwise, additional graph nodes will be created to get `num_classes` value :
<img width="352" alt="model_node" src="https://user-images.githubusercontent.com/10177297/115424748-a57f8280-a207-11eb-9c02-4e7dc9ea5975.PNG">